packages/@react-aria/numberfield/src/useNumberField.ts

```diff
@@ -220,8 +220,12 @@ export function useNumberField(props: NumberFieldProps, state: NumberFieldState,
     {
       onWheel,
       // override the spinbutton role, we can't focus a spin button with VO
-      role: 'textfield',
-      'aria-roledescription': formatMessage('Spin button number field')
+      role: null,
```

Comment:
Wonder if we should just make `useSpinButton` not return a `role` at all? It seems we have been overriding it every time anyway...

Reply:
yeah, we could do that and just put a note in it that VO can't focus it correctly and to apply it again once that works